icevision/models/mmdet/models/retinanet/backbones.py

```diff
@@ -0,0 +1,72 @@
+__all__ = [
+    "r50_caffe_fpn_1x",
+    "r50_fpn_1x",
+    "r50_fpn_2x",
+    "r101_caffe_fpn_1x",
+    "r101_fpn_1x",
+    "r101_fpn_2x",
+    "x101_32x4d_fpn_1x",
+    "x101_32x4d_fpn_2x",
+    "x101_64_4d_fpn_1x",
+    "x101_64x4d_fpn_2x",
+]
+
+r50_caffe_fpn_1x = {
```

Comment:
Sorry for not pointing this out earlier, but I think a better idea would be to use a `dataclass` instead of a dict:
```python
@dataclass
class MMDetBackboneConfig:
    model_name: str
    cfg_filepath: str
    weights_url: str
```